src/main/java/ladder/domain/Ladder.java

```diff
@@ -20,6 +21,37 @@ private void validate(int height) {
         }
     }
 
+    private void initPlayersPosition(Players players, HashMap<Player, Integer> playersPosition){
+        List<Player> playersList = players.players();
+        for(int i = 0; i < playersList.size(); i++){
+            playersPosition.put(playersList.get(i), i);
+        }
+    }
+
+    private void movePlayersPosition(HashMap<Player, Integer> playersPosition){
+        List<Line> lines = this.lines.lines();
+        for(int i = 0; i < lines.size(); i++){
+            move(lines.get(i).points(), playersPosition);
+        }
```

Comment:
```suggestion
             IntStream.range(0, lines.size()).forEach(i -> move(lines.get(i).points(), playersPosition));
```
IntStream.range사용해서 위에처럼 표현이 가능하네요 :)

Reply:
lotto단계를 진행할 때는 forEach를 쓰면 성능상 안 좋다는 피드백을 받았었는데
유라님께서는 for문보다는 forEach를 쓰는걸 권장하시나요?
[이전 로또 단계 피드백](https://github.com/next-step/java-lotto/pull/1331#discussion_r602696327)

Reply:
아하 이전 로또 피드백에서 foreach관련 피드백을 받으셨군요. 
개인적으로는 for문을 지양하긴 합니다. 

현재 미션에서 성능도 중요하지만 
프로그래밍 요구사항의 스트림을 사용하여 구현하라는 부분이 있어서 의견을 드린 부분이며 꼭 고치치 않으셔도 됩니다 
참고 정도로만 ㅎㅎ 확인해주세요!